Source/scrollrt.cpp

```diff
@@ -761,21 +764,33 @@ static void scrollrt_draw_dungeon(int sx, int sy, int dx, int dy)
 	}
 }
 
+/**
+ * @brief Macro for shifting the view area along the logical grid
+ *        Note: this won't allow you to shift between even and odd rows
+ * @param h Shift the screen left or right
+ * @param y Shift the screen up or down

```

Comment:
Should this be `v` instead of `y`?

```suggestion
 * @param v Shift the screen up or down
```

Reply:
It's horizontal and vertical. I wanted to avoid height/width, x/y since it's moving diagonally on the grid, much like a bishop in chess.

I think I will change it to a function `void ShiftView(int *x, int *y, int h, int v)` so that it is easier to reuse for calculating the grid position of the mouse (this is almost done except for two odd exceptions at specific resolutions).